bundles/org.openhab.binding.shelly/README.md

```diff
@@ -97,21 +105,26 @@ Also check out the [Shelly Manager](doc/ShellyManager.md), which
 
 The binding has the following configuration options:
 
-| Parameter      |Description                                                       |Mandatory|Default                                         |
-|----------------|------------------------------------------------------------------|---------|------------------------------------------------|
-| defaultUserId  |Default user id for HTTP authentication when not set in the Thing |    no   |admin                                           |
-| defaultPassword|Default password for HTTP authentication when not set in the Thing|    no   |admin                                           |
-| autoCoIoT      |Auto-enable CoIoT events when firmware 1.6+ is enabled.           |    no   |true                                            |
+### Generation 1
+
+| Parameter      |Description                                                         |Mandatory|Default                                         |
+|----------------|--------------------------------------------------------------------|---------|------------------------------------------------|
+| defaultUserId  |Default user id for HTTP authentication when not set in the Thing   |    no   |admin                                           |
+| defaultPassword|Default password for HTTP authentication when not set in the Thing  |    no   |admin                                           |
+| autoCoIoT      |Auto-enable CoIoT events when firmware 1.6+ is enabled (Gen1 only). |    no   |true                                            |
+
+`defaultUserId` and `defaultPassword:` will be used by the binding if device protection is enabled. However, the Plus/Pro devices have a fixed user id àdmin`. Nevertheless the binding provide that option to allow a mixed operation of Gen 1 and 2 devices in the same installation having same defaults.
 
-The binding defaults to CoIoT events when firmware 1.6 or newer is detected. CoIoT provides near-realtime updates on device status changes.
-This mode also overrules event settings in the Thing configuration. 
+`Generation 1`: The binding defaults to CoIoT events when firmware 1.6 or newer is detected. CoIoT provides near-realtime updates on device status changes.
+This mode also overrules event settings in the Thing configuration. Disabling this feature allows granular control, which event types will be used. This is also required when the Shelly devices are not located on the same IP subnet (e.g. using a VPN). In this case autoCoIoT should be disabled, CoIoT events will not work, because the underlying CoAP protocol is based on Multicast IP, which usually doesn't passes a VPN or routed network.
```

Comment:
If you put each sentence in a separate line, reading the diffs would be easier. Please check all new additions.

Reply:
changed